pkg/reconciler/revision/revision.go

```diff
@@ -54,7 +54,7 @@ import (
 	"github.com/knative/pkg/controller"
 	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
 	listers "github.com/knative/serving/pkg/client/listers/serving/v1alpha1"
-	reconciler "github.com/knative/serving/pkg/controller"
```

Comment:
this was a hit at the correct name 😄

Reply:
They all were.  I didn't want to import `knative/pkg/controller` as like `commoncontroller` only to rename our `controller` package (like this).  `knative/pkg` provides `controller` and `knative/serving` providers `reconciler`.  Feels cleaner to me.